docs/using.md

```diff
@@ -54,12 +63,30 @@ Repeat the process for the following OCI variables:
 | dimensionKey    | `dimensions($region, $compartment, $namespace, $metric, $resourcegroup)`                     |
 | dimensionValue  | `dimensionOptions($region,$compartment,$namespace,$metric,$dimensionKey,$resourcegroup)` |
 
+In case plugin is configured to operate with multitenancy support, use the following OCI variables:
+
+| Name            | Query                                                                     |
+| --------------- | ------------------------------------------------------------------------- |
+| tenancy          | `tenancies()`                                                               |
+| region          | `regions($tenancy)`                                                               |
+| compartment     | `compartments($region,$tenancy)`                                                          |
```

Comment:
For compartments() function having the region as a parameter in multi-tenancy mode is mandatory due to the fact that you may have your plugin configured to address different tenancies which resides in different region, so I need the two parameter to list the compartments. Without the region parameter I could not make it works. $tenancy is set to be the last parameter mainly in order to maintain the retrocompatibility of the regular expression and to to let these functions to work flawlessy in both single tenancy and multi tenancy scenarios:
`    if (resourcegroupQuery) {
      let target = {
        region: removeQuotes(this.getVariableValue(resourcegroupQuery[1])),
        compartment: removeQuotes(this.getVariableValue(resourcegroupQuery[2])),
        namespace: removeQuotes(this.getVariableValue(resourcegroupQuery[3])),
        tenancy: removeQuotes(this.getVariableValue(resourcegroupQuery[4])),
      };`

Reply:
However I maybe found a solution which does not impact too much the single tenancy mode. I am testing it right now

Reply:
It is done, but need to be tested for both single tenancy and multi tenancy mode, since I modified how the "target" is built after the template vars in datasource.js